### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by vIremAydin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,8 +14,11 @@ system reports like finding the most popular events in a month based on the numb
 people who attended etc.
 
 We have used HTML and CSS for the pages, PHP for connecting them to the database, and SQL for the database itself.
+h.
```
Comment (Ece-Kahraman) [inline README.md:17]: napıyon

Comment (Ece-Kahraman) [inline README.md:17]: eyo

Comment (vIremAydin) [inline README.md:17]: iyidir canım

Comment (Ece-Kahraman) [reply README.md:17]: sil bunu

**Comment by Ece-Kahraman** [comment]
sen sen

File: README.md [hunk 2]
```diff
@@ -14,8 +14,11 @@ system reports like finding the most popular events in a month based on the numb
```
Comment (Ece-Kahraman) [inline README.md:13]: aaaa

File: README.md [hunk 3]
```diff
@@ -14,8 +14,11 @@ system reports like finding the most popular events in a month based on the numb
 people who attended etc.
```
Comment (Ece-Kahraman) [inline README.md:14]: aaaa

File: README.md [hunk 4]
```diff
@@ -14,8 +14,11 @@ system reports like finding the most popular events in a month based on the numb
...
 
 We have used HTML and CSS for the pages, PHP for connecting them to the database, and SQL for the database itself.
```
Comment (Ece-Kahraman) [inline README.md:16]: aaaa

File: README.md [hunk 5]
```diff
@@ -14,8 +14,11 @@ system reports like finding the most popular events in a month based on the numb
 people who attended etc.
```
Comment (Ece-Kahraman) [inline README.md:15]: ISSUE (BLOCKING): change required

Comment (Ece-Kahraman) [inline README.md:15]: ISSUE (IF-MINOR): hello

**Review by Ece-Kahraman — APPROVED** [review]
hello

File: README.md (hunk 5 above)
Comment (Ece-Kahraman) [inline README.md:15]: ISSUE (IF-MINOR): hello

**Review by Ece-Kahraman — APPROVED** [review]
hello

File: README.md (hunk 4 above)
Comment (Ece-Kahraman) [inline README.md:16]: THOUGHT (IF-MINOR): hello

**Review by Ece-Kahraman — CHANGES_REQUESTED** [review]
hello

**Pull request merged**